.github/CODEOWNERS

```diff
@@ -8,5 +8,5 @@
 # global owners are only requested if there isn't a more specific
 # codeowner specified below. For this reason, the global codeowners
 # are often repeated in package-level definitions.
-* @liamsi @MSevey @evan-forbes @renaynay @Wondertan @tzdybal @Bidon15
+* @liamsi @MSevey @Bidon15 @jrmanes @sysrex @smuu
```

Comment:
Can you update this to be 
- Ismail and me as defaults 
- the GitHub testing team and myself for .github directory? 

Having the split out like that will make it so Ismail isn't being bothered with the workflow and action PRs. 

Also you can target the actual GitHub team instead of individuals so we don't need to worry about updating this potential ever again.

Reply:
> Also you can target the actual GitHub team instead of individuals so we don't need to worry about updating this potential ever again

OMG WE CAN DO THIS?

Reply:
yes. We need to figure out the write access @evan-forbes 

https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization